src/sentry/replays/usecases/query/configs/scalar.py

```diff
@@ -0,0 +1 @@
+"""Aggregate query filtering configuration module."""
```

Comment:
this file meant to be empty?

Reply:
No I wanted to add the sub-query configuration here but I haven't yet.  I thought about doing it in a follow-up PR.  What are your thoughts?  Add the sub-query now or later?

Reply:
ah got it. can be done in a follow up. 👍🏼